cvat/apps/training/jobs.py

```diff
@@ -0,0 +1,180 @@
+from collections import OrderedDict
+from typing import List
+
+from cacheops import cache
+from django_rq import job
```

Comment:
@dmitryagapov Seems there is a reason of failing unitests on CI;
We use fakeredis package to run unittests instead of real redis instance and looks like this decorator try to connect to the redis server directly. I will investigate this issue further.